src/components/Home.jsx

```diff
@@ -58,20 +62,17 @@ const SubHero = (props) => {
   );
 };
 
-const Party = (props) => {
-  const [currentHero, setCurrentHero] = useState(
-    props.party[props.party.length - 1]
-  );
+const CardExamples = (props) => {
   return (
     <div id="party">
-      {props.party.map((hero) => {
-        return <HeroCard key={hero.name} {...hero} />;
+      {heroList.map((hero) => {
+        return <PlayingCard key={hero.name} {...hero} />;
       })}
     </div>
   );
 };
 
-const HeroCard = (props) => {
+const PlayingCard = (props) => {
```

Comment:
Why playing card? 

I feel hero card and element card worked well for naming conventions, but maybe I'm missing something please elaborate?